source/tests/cppexpose-test/CMakeLists.txt

```diff
@@ -92,9 +92,19 @@ target_compile_definitions(${target}
 # Compile options
 # 
 
+# MSVC compiler options
+if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "MSVC")
+    set(TARGET_COMPILE_OPTIONS ${DEFAULT_COMPILE_OPTIONS}
+    PRIVATE
+        /bigobj
```

Comment:
@sbusch42 Do we have a pattern/template/recipe in cmake-init on handling ```bigobj``` for MSVC?